README.md

```diff
@@ -57,7 +57,7 @@ The specification and code is licensed under the Apache 2.0 license found in the
 
 ## Code of Conduct
 
-Participation in the OpenContainers community is governed by [OpenContainer's Code of Conduct](code-of-conduct.md).
+Participation in the OpenContainers community is governed by [OpenContainer's Code of Conduct](https://github.com/opencontainers/tob/blob/master/code-of-conduct.md).
```

Comment:
Same question I was going to ask on the image-spec PR

On Thu, Apr 7, 2016, 21:39 Qiang Huang <notifications@github.com> wrote:

> In README.md
> <https://github.com/opencontainers/runtime-spec/pull/375#discussion_r58970830>
> :
>
> > @@ -57,7 +57,7 @@ The specification and code is licensed under the Apache 2.0 license found in the
> >
> >  ## Code of Conduct
> >
> > -Participation in the OpenContainers community is governed by [OpenContainer's Code of Conduct](code-of-conduct.md).
> > +Participation in the OpenContainers community is governed by [OpenContainer's Code of Conduct](https://github.com/opencontainers/tob/blob/master/code-of-conduct.md).
>
> Then shall we still keep the code-of-conduct.md in rutime-spec and keep
> it syncing with the tob one?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/375/files/b05cbff7a7a57147a626ab2665a42f35c0e6ee09#r58970830>
>